include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -275,7 +219,17 @@ struct is_hetero_legacy_trait<Iter, ::std::enable_if_t<Iter::is_hetero::value>>
 };
 
 template <typename Iter>
-inline constexpr bool is_sycl_iterator_v = is_sycl_iterator<Iter>::value || is_hetero_legacy_trait<Iter>::value;
+using is_sycl_or_hetero_iterator = std::disjunction<is_sycl_iterator<Iter>, is_hetero_legacy_trait<Iter>>;
```

Comment:
Am I right that "sycl iterator" is a special case of more general term "hetero iterator"?
If so, I would suggest more general name `is_hetero_iterator`.

Reply:
suggestion taken.